.github/workflows/post-merge.yml

```diff
@@ -8,6 +8,7 @@ on:
 permissions:
   contents: read
   id-token: write
+  packages: read
```

Comment:
I think you'll also need to add `packages:read` to the `build` job too (below line 24) and the `run-tests` job (below line 29).

I've also added this change to our commit so you could just replace the whole commit with ours.